packages/jss-plugin-nested/src/index.js

```diff
@@ -1,7 +1,12 @@
 import warning from 'tiny-warning'
 
-const separatorRegExp = /\s*,\s*/g
-const parentRegExp = /&/g
+// cssfn:
+import {
+  parseSelectors,
+  flatMapSelectors,
+  selectorsToString,
+}                           from '@cssfn/css-selector'
```

Comment:
I am afraid we can't do this, its a huge bundle-size increase

Reply:
We need a solution that is super minimalistic

Reply:
Wow did you write the selector parser to fix this bug? If so huge props in any case!
But jss is runtime-executed, it gets bundled with a client-side app, the only way forward is to reduce the size, even potentially remove features to make it small.

Reply:
@heyymarco wow dude! thats some serious work!